pkgs/by-name/dp/dprint/plugins/default.nix

```diff
@@ -11,6 +15,7 @@ let
       updateUrl,
       license ? lib.licenses.mit,
       maintainers ? [ lib.maintainers.phanirithvij ],
+      formattedFile,
```

Comment:
Setting the default filename with `initConfig.fileExtensions` blocked with nixpkgs-vet, and I agree it might be unstable. So simply added the typical filenames for each plugin.

Reply:
sent a pr against your fork, updating the python codebase to generate these.

Reply:
Finally I use `--allow-no-files` for simplicity. However I keep their work in co-author.